src/@types/spid-passport/index.d.ts

```diff
@@ -6,6 +6,7 @@ declare module "spid-passport";
 
 declare class SpidStrategy {
   // TODO: define types for constructor (see errors in spidStrategy.ts)
-  public logout(req: any, callback?: (any, any) => void): void;
+  // tslint:disable-next-line:no-any
```

Comment:
disabling `no-any` should be a last resort for very special cases - in this case I'm sure you'll be able to define more specific types